### ISSUE TITLE
Json to JsonLd: consistent naming

### ISSUE THREAD

**Issue by jindrichmynarz** [linkedpipes/etl#432]
To preserve consistent naming "Json to JsonLD" component should be named "JSON to JSON-LD", as in JSON-LD operations. This name is already used in the [documentation](http://etl.linkedpipes.com/components/t-jsontojsonld), but the old name is shown in the pipeline editor.

**Comment by jindrichmynarz** [comment]
Build of  t-jsonToJsonLd fails tests in 925e05d5ddfec1053cb0b32454ba6c8a1b95991d with the following exception:

```
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.649 sec <<< FAILURE! - in com.linkedpipes.plugin.transformer.jsontojsonld.ValidationTest
verifyConfigurationDescription(com.linkedpipes.plugin.transformer.jsontojsonld.ValidationTest)  Time elapsed: 0.64 sec  <<< ERROR!
com.fasterxml.jackson.databind.JsonMappingException: Unexpected character ('J' (code 74)): was expecting comma to separate OBJECT entries
 at [Source: java.io.BufferedReader@3e84448c; line: 5, column: 61]
 at [Source: java.io.BufferedReader@3e84448c; line: 5, column: 58] (through reference chain: java.util.ArrayList[0])
	at com.fasterxml.jackson.core.JsonParser._constructError(JsonParser.java:1586)
	at com.fasterxml.jackson.core.base.ParserMinimalBase._reportError(ParserMinimalBase.java:521)
	at com.fasterxml.jackson.core.base.ParserMinimalBase._reportUnexpectedChar(ParserMinimalBase.java:450)
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser._skipComma(ReaderBasedJsonParser.java:2204)
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser.nextFieldName(ReaderBasedJsonParser.java:863)
	at com.fasterxml.jackson.databind.deser.std.UntypedObjectDeserializer$Vanilla.mapObject(UntypedObjectDeserializer.java:654)
	at com.fasterxml.jackson.databind.deser.std.UntypedObjectDeserializer$Vanilla.deserialize(UntypedObjectDeserializer.java:496)
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:277)
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:249)
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:26)
	at com.fasterxml.jackson.databind.ObjectMapper._readValue(ObjectMapper.java:3761)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:2042)
	at com.fasterxml.jackson.core.JsonParser.readValueAs(JsonParser.java:1511)
	at com.github.jsonldjava.utils.JsonUtils.fromReader(JsonUtils.java:119)
	at com.github.jsonldjava.utils.JsonUtils.fromInputStream(JsonUtils.java:97)
	at com.github.jsonldjava.utils.JsonUtils.fromInputStream(JsonUtils.java:77)
	at org.eclipse.rdf4j.rio.jsonld.JSONLDParser.parse(JSONLDParser.java:68)
	at org.eclipse.rdf4j.rio.Rio.parse(Rio.java:256)
	at org.eclipse.rdf4j.rio.Rio.parse(Rio.java:182)
	at com.linkedpipes.etl.test.suite.TestConfigurationDescription.readDefinition(TestConfigurationDescription.java:197)
	at com.linkedpipes.etl.test.suite.TestConfigurationDescription.readDescriptorReference(TestConfigurationDescription.java:188)
	at com.linkedpipes.etl.test.suite.TestConfigurationDescription.validateDescriptorsReference(TestConfigurationDescription.java:175)
	at com.linkedpipes.etl.test.suite.TestConfigurationDescription.test(TestConfigurationDescription.java:53)
	at com.linkedpipes.plugin.transformer.jsontojsonld.ValidationTest.verifyConfigurationDescription(ValidationTest.java:12)


Results :

Tests in error:
  ValidationTest.verifyConfigurationDescription:12 » JsonMapping Unexpected char...

Tests run: 1, Failures: 0, Errors: 1, Skipped: 0
```

**Comment by jindrichmynarz** [comment]
Fixed in 4519bdc9b3a6993ae706152d6376c234e1835233.
